ci/kind/kind-setup.sh

```diff
@@ -563,6 +608,21 @@ while [[ $# -gt 0 ]]
       VLAN_ID="$2"
       shift 2
       ;;
+    --ipam-vlan-subnets)
```

Comment:
If `--ipam-vlan-subnets` or `--ipam-vlan-ids` is not defined when `--flexible-ipam` is set, should we return error message?

Reply:
yes, I added that condition. thanks!